client/server/pages/index.js

```diff
@@ -155,7 +155,7 @@ function getDefaultContext( request, entrypoint = 'entry-main' ) {
 		badge: false,
 		lang,
 		entrypoint: request.getFilesForEntrypoint( entrypoint ),
-		manifest: request.getAssets().manifests.manifest,
+		manifest: request.getAssets().manifests.join( '\n' ),
```

Comment:
The `manifests` object seems to only ever have one key: `runtime`. Are there scenarios where there are multiple.

Concatenating scripts with a simple newline is not 100% reliable. I've seen some funny examples in the past, and if I remember correctly, they were not theoretical, but happened in practice. Two `<script>` tags next to each other would be safer.